Shipr-Helm/templates/ingress.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: shipr-ingress
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: /
+spec:
+  rules:
+  - host: shipr.local
+    http:
+      paths:
+      - path: /frontend
+        pathType: Prefix
+        backend:
+          service:
+            name: shipr-frontend-service
+            port:
+              number: 80
+      - path: /payment
+        pathType: Prefix
+        backend:
+          service:
+            name: shipr-payment-service
+            port:
+              number: 80
+      - path: /inventory
+        pathType: Prefix
+        backend:
+          service:
+            name: shipr-inventory-service
+            port:
+              number: 80
```

Comment:
Do you think any property can be templated here?

Reply:
Yes, there is many properties that can be templated here like host, paths and ports. you can change that according to the environment through vaules file.

Reply:
Please template it then. Any reason it is not being templated?

Reply:
I think you mistakenly removed ingress from helm chart. It is part of the requirements for this project.

Reply:
Added